src/asound/20-bluealsa.conf

```diff
@@ -47,6 +47,22 @@ pcm.bluealsa {
 			name defaults.bluealsa.profile
 		}
 	}
+	@args.CODEC {
+		type string
+		default ""
```

Comment:
Why there is no `name default.bluealsa....`?

Reply:
I can certainly add that if you prefer. Personally, I am uncomfortable with using the empty string that way. Perhaps an alternative would be to introduce a 'special' value such as "unchanged" as the default, and modify the plugin code to handle that. The other params could also be modified the same way.

... Now I wish I'd thought  of this idea originally, I think it is a more user-friendly approach, do you agree?

Reply:
> Perhaps an alternative would be to introduce a 'special' value such as "unchanged" as the default.

Yes, that seems to be a good idea. The problem with not having "name default" is the consistency with other arguments. Since there are "default.bluealsa..." names for other options, one might expect that all options might be configured that way.